libs/transfer/src/lib/views/make-transfer-success-view/make-transfer-success-view.component.ts

```diff
@@ -3,10 +3,10 @@ import { ActivatedRoute, Router } from '@angular/router';
 import { MakeTransferJourneyState } from '../../services/make-transfer-journey-state.service';
 
 @Component({
-  templateUrl: 'make-transfer-success-view.component.html'
+  templateUrl: 'make-transfer-success-view.component.html',
 })
 export class MakeTransferSuccessViewComponent {
-  title = this.route.snapshot.data.title;
+  title = this.route.snapshot.data['title'];
```

Comment:
Dot notation, please.

Reply:
I changed because property 'title' comes from an index signature, so it must be accessed with ['title'].ts(4111)

I can disable noPropertyAccessFromIndexSignature if needed.

Reply:
It's ok.